pkg/manager/member/startscript/v2/pd_start_script.go

```diff
@@ -42,8 +42,8 @@ type PDStartScriptModel struct {
 
 // PDMSStartScriptModel contain fields for rendering PD Micro Service start script
 type PDMSStartScriptModel struct {
-	PDMSDomain string
-	PDMSName   string
+	PDDomain       string
```

Comment:
in fact, is it the domain of the real main PD process, right?

Reply:
just want to reuse script `pdWaitForDnsOnlySubScript`

Reply:
yes, changed